samples/sampler_wrap.html

```diff
@@ -0,0 +1,241 @@
+<!DOCTYPE html>
+<!-- Ported from the OpenGL Samples Pack https://github.com/g-truc/ogl-samples -->
+<html lang="en">
+
+<head>
+    <title>WebGL 2 Samples - sampler_wrap</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="utility.js"></script>
+</head>
+
+<body>
+    <div id="info">WebGL 2 Samples - sampler_wrap</div>
+
+    <script id="vs" type="x-shader/x-vertex">
+        #version 300 es
+        precision highp float;
+        precision highp int;
+
+        layout(std140, column_major) uniform;
+
+        uniform mat4 MVP;
+
+        layout(location = 0) in vec2 Position;
+        layout(location = 4) in vec2 Texcoord;
```

Comment:
Likewise, this is called `texcoord` in the other samples.  It should probably be `texCoord` in all.  Also, consider avoiding abbreviations as much as possible: `textureCoordinates`.